yt_dlp/extractor/c13cl.py

```diff
@@ -0,0 +1,13 @@
+from .common import InfoExtractor
+
+
+class C13ClIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?13\.cl/en-vivo'
+
+    def _real_extract(self, url):
+        display_id = 'C13'
+        webpage = self._download_webpage(url, display_id)
+
+        stream_url = self._search_regex(r'<div\s+.*id=(?:"|\')player(?:"|\')[^>]+><iframe\s+.*src=(?:"|\')([^?]+)', webpage, 'stream_url')
```

Comment:
imo. since you don't do any metadata extraction here, you should just use `_EMBED_REGEX` in the RudoVideo extractor